rllib/scripts.py

```diff
@@ -1,48 +1,330 @@
+        env=env,
+        local_mode=local_mode,
+        render=render,
+        video_dir=video_dir,
+        steps=steps,
+        episodes=episodes,
+        out=out,
+        config=config,
+        save_info=save_info,
+        use_shelve=use_shelve,
+        track_progress=track_progress,
     )
-    rollout_parser = evaluate.create_parser(
-        lambda **kwargs: subcommand_group.add_parser("rollout", **kwargs)
+
+
+@app.command()
+def rollout(
+    checkpoint: str = typer.Argument(None, help=eval_help.get("checkpoint")),
+    run: str = typer.Option(..., "--run", "-r", help=eval_help.get("run")),
+    env: str = typer.Option(..., "--env", "-e", help=eval_help.get("env")),
+    local_mode: bool = typer.Option(False, help=eval_help.get("local_mode")),
+    render: bool = typer.Option(False, help=eval_help.get("render")),
+    video_dir: str = typer.Option(None, help=eval_help.get("video_dir")),
+    steps: int = typer.Option(10000, help=eval_help.get("steps")),
+    episodes: int = typer.Option(0, help=eval_help.get("episodes")),
+    out: str = typer.Option(None, help=eval_help.get("out")),
+    config: str = typer.Option("{}", help=eval_help.get("config")),
+    save_info: bool = typer.Option(False, help=eval_help.get("save_info")),
+    use_shelve: bool = typer.Option(False, help=eval_help.get("use_shelve")),
+    track_progress: bool = typer.Option(False, help=eval_help.get("track_progress")),
+):
+    """This command is deprecated.
+    Please use the "evaluate" command with the same arguments."""
+
+    from ray.rllib.utils.deprecation import deprecation_warning
+
+    deprecation_warning(old="rllib rollout", new="rllib evaluate", error=False)
+
+    return evaluate(
+        checkpoint=checkpoint,
+        run=run,
+        env=env,
+        local_mode=local_mode,
+        render=render,
+        video_dir=video_dir,
+        steps=steps,
+        episodes=episodes,
+        out=out,
+        config=config,
+        save_info=save_info,
+        use_shelve=use_shelve,
+        track_progress=track_progress,
     )
-    options = parser.parse_args()
-
-    if options.command == "train":
-        train.run(options, train_parser)
-    elif options.command == "evaluate":
-        evaluate.run(options, evaluate_parser)
-    elif options.command == "rollout":
-        deprecation_warning(old="rllib rollout", new="rllib evaluate", error=False)
-        evaluate.run(options, rollout_parser)
-    else:
-        parser.print_help()
+
+
+@app.callback()
+def main_helper():
+    """
+    Welcome to the\n\n
+
+    ██████╗ ██╗     ██╗     ██╗██████╗\n
+    ██╔══██╗██║     ██║     ██║██╔══██╗\n
+    ██████╔╝██║     ██║     ██║██████╔╝\n
+    ██╔══██╗██║     ██║     ██║██╔══██╗  command\n
+    ██║  ██║███████╗███████╗██║██████╔╝  line\n
+    ╚═╝  ╚═╝╚══════╝╚══════╝╚═╝╚═════╝   interface\n\n
+
+    Example usage for training:\n
+        rllib train --run DQN --env CartPole-v1\n\n
+
+    Example usage for evaluate (aka: "rollout"):\n
```

Comment:
Typo? "Example usage for evaluation ..."

Reply:
I copied this from the existing one :D